### PR TITLE
UHM-4810: Create "hiv_patient_program" table in PETL

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

File: configuration/pih/petl/jobs/hiv/program/source.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  outcome_concept_id int,
+  outcome varchar(255)
+);
+
+INSERT into temp_patient_program (patient_program_id, patient_id, date_enrolled, date_completed, location_id,outcome_concept_id)
+select patient_program_id, patient_id, date_enrolled, date_completed,location_id,outcome_concept_id from patient_program where voided=0;
+
+update temp_patient_program
+set location = location_name(location_id),
+    outcome = concept_name(outcome_concept_id, 'en');
```
Comment (mogoodrich) [inline configuration/pih/petl/jobs/hiv/program/source.sql:23]: Hardcoding to English for now?

**Review by mseaton — APPROVED** [review]
LGTM

File: configuration/pih/petl/jobs/hiv/program/source.sql (hunk 1 above)
Comment (jmbabazi) [reply configuration/pih/petl/jobs/hiv/program/source.sql:23]: Thanks @mogoodrich , LGTM

**Review by jmbabazi — APPROVED** [review]
LGTM @mogoodrich

**Pull request merged**